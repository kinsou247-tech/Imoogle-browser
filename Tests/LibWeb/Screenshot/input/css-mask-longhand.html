<!DOCTYPE html>
<link rel="match" href="../expected/css-mask-longhand.html" />
<div class="masked"></div>

<style>
    .masked {
        display: block;
        width: 200px;
        height: 200px;
        mask: url("../data/smiley.png");
        background-color: blue;
    }
</style>
<!-- FIXME: Workaround to ensure CSS background-image is loaded before taking screenshot: https://github.com/kinsou247-tech/Imoogle-browser/issues/3448 -->
<img src="../data/smiley.png" />
