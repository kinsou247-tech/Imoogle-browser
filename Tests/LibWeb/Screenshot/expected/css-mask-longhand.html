<!DOCTYPE html>
<div class="masked"></div>

<style>
    .masked {
        display: block;
        width: 200px;
        height: 200px;
        mask-image: url("../data/smiley.png");
        background-color: blue;
    }
</style>
<!-- FIXME: Workaround to ensure CSS background-image is loaded before taking screenshot: https://github.com/kinsou247-tech/Imoogle-browser/issues/3448 -->
<img src="../data/smiley.png" />
